src/lightning/fabric/loggers/csv_logs.py

```diff
@@ -158,9 +158,10 @@ def _get_next_version(self) -> int:
             return 0
 
         existing_versions = []
-        for d in self._fs.listdir(root_dir, detail=False):
-            name = d[len(root_dir) + 1 :]  # removes parent directories
-            if self._fs.isdir(d) and name.startswith("version_"):
+        for d in self._fs.listdir(root_dir):
```

Comment:
lets add a test for this

Reply:
I added a test for this case. And I can confirm that the test failed before, and passed after this fix.